pkg/plugins/runtime/k8s/webhooks/injector/injector.go

```diff
@@ -404,6 +301,8 @@ func (i *KumaInjector) NewSidecarContainer(
 	}
 
 	container.Name = k8s_util.KumaSidecarContainerName
+	container.Resources.Requests[kube_core.ResourceEphemeralStorage] = pointer.Deref(kube_api.NewScaledQuantity(10, kube_api.Mega))
+	container.Resources.Limits[kube_core.ResourceEphemeralStorage] = pointer.Deref(kube_api.NewScaledQuantity(10, kube_api.Mega))
```

Comment:
should this be added only for injected sidecar or also for builtin gateway? CC @michaelbeaumont If for both, it's better to place it in `NewContainer()` func

Reply:
I've moved this into `factory.go` so it can be shared for both sidecars and gateways